base/fastmath.jl

```diff
@@ -124,6 +125,27 @@ function make_fastmath(symb::Symbol)
 end
 make_fastmath(expr) = expr
 
+"""
+    @fastmath expr
+
+Executes a transformed version of the expression, which calls functions that
```

Comment:
Perhaps `Executes` -> `Execute` in keeping with the documentation guidelines? :)